File: Data Structures & Algorithms/two-integer-sum/submission-3.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}
        for i, num in enumerate(nums):
            difference = target - num
            if difference in hashMap:
                return [min(i, hashMap[difference]), max(i, hashMap[difference])]
            else:
                hashMap[num] = i

File: Data Structures & Algorithms/majority-element/submission-3.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int result = 0;
        int count = 0;

        for (int num : nums) {
            if (count == 0) {
                result = num;
                count++;
                continue;
            } 
            result == num ? count++ : count--;
        }    
        return result;
    }
};